dapps/js-glue/build.rs

```diff
@@ -8,12 +8,11 @@
 
 // Parity is distributed in the hope that it will be useful,
 // but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
```

Comment:
Regarding the extra space, I think it's actually intentional? Reading from [Wikipedia](https://en.wikipedia.org/wiki/Sentence_spacing) this is something called "English spacing". And I do remember encountering of this spacing strategy in my life.

And in particular, the original GPL texts uses English spacing. https://www.gnu.org/licenses/gpl.txt